tests/rancher/rancher_ui_selenium.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Web browser UI test for rancher container
+# Maintainer: George Gkioulis <ggkioulis@suse.com>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use x11utils 'ensure_unlocked_desktop';
+
+sub run {
+    my ($self) = @_;
+    
+    select_console('x11', await_console => 0);
+    ensure_unlocked_desktop();
```

Comment:
```suggestion
    $self->select_serial_terminal;
```

Reply:
Thanks for your comments. We used a mix of them.